apps/root/src/pages/earn/components/wizard/components/suggested-strategies/index.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { Strategy, Token } from 'common-types';
+import styled from 'styled-components';
+import {
+  BackgroundPaper,
+  Button,
+  ContainerBox,
+  Grid,
+  Typography,
+  colors,
+  StarEmoji,
+  KeyboardArrowRightIcon,
+} from 'ui-library';
+import DataCards, { DataCardVariants } from '@pages/strategy-guardian-detail/vault-data/components/data-cards';
+import TokenIconWithNetwork from '@common/components/token-icon-with-network';
+import { FormattedMessage } from 'react-intl';
+import useAllStrategies from '@hooks/earn/useAllStrategies';
+import { getIsSameOrTokenEquivalent } from '@common/utils/currency';
+import { intersectionBy, uniqBy } from 'lodash';
+import usePushToHistory from '@hooks/usePushToHistory';
+import { useThemeMode } from '@state/config/hooks';
+
+interface SugestedStrategyCardProps {
+  strategy: Strategy;
+}
+
+const StyledPaper = styled(BackgroundPaper).attrs({ variant: 'outlined' })`
+  ${({ theme: { palette, spacing } }) => `
+    padding: ${spacing(6)};
+    background: ${colors[palette.mode].background.tertiary};
+    box-shadow: ${colors[palette.mode].dropShadow.dropShadow300};
+  `}
+`;
+
+const SuggestedStrategyCard = ({ strategy }: SugestedStrategyCardProps) => {
+  const pushToHistory = usePushToHistory();
+  const handleViewStrategy = () => {
+    pushToHistory(`/earn/${strategy.network.chainId}/vaults/${strategy.id}`);
+  };
+
+  return (
+    <StyledPaper>
+      <ContainerBox flexDirection="column" alignItems="stretch" gap={5}>
+        <ContainerBox justifyContent="space-between" alignItems="center">
+          <ContainerBox gap={2} alignItems="center">
+            <TokenIconWithNetwork token={strategy.asset} />
+            <Typography variant="bodySmallRegular">{strategy.asset.symbol}</Typography>
+          </ContainerBox>
+          <Typography variant="bodySmallRegular">{strategy.farm.name}</Typography>
+        </ContainerBox>
+        <DataCards strategy={strategy} dataCardsGap={2} variant={DataCardVariants.Wizard} />
+        <Button
+          variant="text"
+          color="primary"
+          onClick={handleViewStrategy}
+          fullWidth
+          endIcon={<KeyboardArrowRightIcon />}
+        >
+          <FormattedMessage description="earn.wizard.suggestedStrategies.button" defaultMessage="View Vault" />
+        </Button>
+      </ContainerBox>
+    </StyledPaper>
+  );
+};
+
+interface SuggestedStrategiesProps {
+  selectedAsset: {
+    chainsWithBalance: number[];
+    token: Token;
+  };
+  selectedReward: {
+    token: Token;
+  };
+}
+const SuggestedStrategies = ({ selectedAsset, selectedReward }: SuggestedStrategiesProps) => {
+  const { strategies } = useAllStrategies();
+  const themeMode = useThemeMode();
+
+  const suggested = React.useMemo<Strategy[]>(() => {
```

Comment:
nice logic! Should we send it to a hook? Seems like we may be able to re-use this at some point if we add another way of seeing suggested strats